h/views/api_flags.py

```diff
@@ -16,8 +20,18 @@
             effective_principals=security.Authenticated)
 def create(context, request):
     annotation = _fetch_annotation(context, request)
+
     svc = request.find_service(name='flag')
     svc.create(request.authenticated_user, annotation)
+
+    group_service = request.find_service(IGroupService)
+    group = group_service.find(annotation.groupid)
+
+    incontext_link = links.incontext_link(request, annotation)
+
+    send_params = flag_notification.generate(request, group.creator.email, incontext_link)
+    mailer.send.delay(*send_params)
```

Comment:
I think it might be nice to move the new code here into a private method to better structure the code, the tests can still be the same as we don't test private functions individually. This method should probably just get the request and the annotation, it can then fetch the group, the in-context link and send off the email.